sql/src/test/java/org/apache/druid/sql/calcite/schema/DruidCalciteSchemaModuleTest.java

```diff
@@ -121,6 +124,9 @@ public void setUp()
           binder.bind(FilteredServerInventoryView.class).toInstance(serverInventoryView);
           binder.bind(SegmentManager.class).toInstance(segmentManager);
           binder.bind(DruidOperatorTable.class).toInstance(druidOperatorTable);
+          binder.bind(CoordinatorClient.class)
+                .annotatedWith(Coordinator.class)
```

Comment:
I don't think `CoordinatorClient` is ever used with an annotation.

Reply:
fair, it's not used anymore